taskbuilder.js

```diff
@@ -129,8 +129,18 @@ function getTask(ops) {
         host,
         actions;
 
+    // alias prod(uction) to the production hostname
     if (ops.host.match(/prod(uction)?/)) {
         host = DDG;
+    // if hostname contains a dot, assume it's a full one
+    // rather than a subdomain
+    } else if (ops.host.indexOf(".") > -1) {
+        host = ops.host;
+    // if there's a port but no dot assume a duckduckgo subdomain with a port
```

Comment:
This will be fine for running things locally, but when we use browsers other than Phantom the request is done from BrowserStack's servers.